doc/book/admin/monitoring/grafana_dashboard.rst

```diff
@@ -0,0 +1,165 @@
+.. _monitoring-grafana_dashboard-page:
+
+===============================================================================
+Grafana dashboard
+===============================================================================
+
+After :ref:`enabling and configuring metrics <monitoring-getting_started>`, you can visualise them using Tarantool Grafana dashboards.
+These dashboards are available as part of
+`Grafana official & community-built dashboards <https://grafana.com/grafana/dashboards>`_:
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 50 50
+        :header-rows: 0
+
+        *   -   Tarantool 3
+            -   `Prometheus <https://grafana.com/grafana/dashboards/21474>`_, `InfluxDB <https://grafana.com/grafana/dashboards/21484>`_
+
+        *   -   Tarantool Cartridge and Tarantool 1.10—2.x
+            -   `Prometheus <https://grafana.com/grafana/dashboards/13054>`_, `InfluxDB <https://grafana.com/grafana/dashboards/12567>`_
+
+        *   -   Tarantool Data Grid 2
+            -   `Prometheus <https://grafana.com/grafana/dashboards/16406>`_, `InfluxDB <https://grafana.com/grafana/dashboards/16405>`_
+
+The Tarantool Grafana dashboard is a ready for import template with basic memory,
+space operations, and HTTP load panels, based on default `metrics <https://github.com/tarantool/metrics>`_
+package functionality.
+
+.. image:: images/Prometheus_dashboard_1.png
+   :width: 30%
+
+.. image:: images/Prometheus_dashboard_2.png
+   :width: 30%
+
+.. image:: images/Prometheus_dashboard_3.png
+   :width: 30%
+
+.. _monitoring-grafana_dashboard-monitoring_stack:
+
+-------------------------------------------------------------------------------
+Prepare a monitoring stack
+-------------------------------------------------------------------------------
+
+Since there are Prometheus and InfluxDB data source Grafana dashboards,
+you can use one of the following:
+
+- `Telegraf <https://www.influxdata.com/time-series-platform/telegraf/>`_
+  as a server agent for collecting metrics, `InfluxDB <https://www.influxdata.com/>`_
+  as a time series database for storing metrics, and `Grafana <https://grafana.com/>`_
+  as a visualization platform.
+- `Prometheus <https://prometheus.io/>`_ as both a server agent for collecting metrics
+  and a time series database for storing metrics, and `Grafana <https://grafana.com/>`_
+  as a visualization platform.
+
+For issues related to setting up Prometheus, Telegraf, InfluxDB, or Grafana instances, refer to the corresponding project's documentation.
+
+.. _monitoring-grafana_dashboard-collect_metrics:
+
+-------------------------------------------------------------------------------
+Collect metrics with server agents
+-------------------------------------------------------------------------------
+
+To collect metrics for Prometheus, first set up metrics output with ``prometheus`` format.
```

Comment:
Seems like third-level headings would be useful here.

Reply:
Totally agree